test/project_v2/api/v1/doc.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Kubernetes authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1 contains API Schema definitions for the crew v1 API group
+// +k8s:openapi-gen=true
+// +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=sigs.k8s.io/kubebuilder/test/project_v2/pkg/apis/crew
+// +k8s:defaulter-gen=TypeMeta
+// +groupName=crew.testproject.org
```

Comment:
https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/simplified-scaffolding.md is designed for handling one Group, but it seem we have multiple groups under `/api`.

Reply:
Aah, completely missed that aspect. Will take a look.

Reply:
Just realized, haven't addressed this one, taking a look now.